rero_ils/modules/patrons/api.py

```diff
@@ -180,8 +180,7 @@ def _validate_emails(self):
         Check if the user has at least one email if the communication channel
         is email.
         """
-        patron = self.get('patron')
-        user = self._get_user_by_user_id(self.get('user_id'))
+        patron, user = self.get('patron'), self.user
         if patron and patron.get('communication_channel') == 'email'\
                 and user.email is None\
```

Comment:
If we use `user` only on this place we can delete `user = self.user` above and write `self.user.email`